.changeset/popular-seahorses-kiss.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-connectors/burner-wallet-connector": minor
+---
+
+Add `Burner Wallet` recovery feature.
+> A dialog will now prompt the user to recover their previous Burner Wallet session.
+
+Also, fix connection status issue when refreshing the page (it was losing the connection status).
```

Comment:
can we split this in 2 changesets ?

1- Add Dialog that will prompt the user to recover his previous Burner Wallet session
2- Fix connection being lost when refreshing the page

Reply:
https://github.com/FuelLabs/fuel-connectors/pull/178/commits/522edde141effa771ea9712fab637d3e28de6b62